backend/internal/utils/validators.go

```diff
@@ -0,0 +1,18 @@
+// internal/utils/validators.go
+package utils
+
+import (
+    "regexp"
+    "strings"
+)
+
+// Check if the string is a valid email address
+func IsEmail(input string) bool {
+    return strings.Contains(input, "@")
```

Comment:
For a more robust email validity checker, we should validate the string using regex:

`^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$`

...representing the email prefix and the top-level domain.